ResearchKit/Charts/ORKGraphChartView.h

```diff
@@ -106,20 +109,6 @@ ORK_AVAILABLE_DECL
 - (NSInteger)graphChartView:(ORKGraphChartView *)graphChartView numberOfPointsForPlotIndex:(NSInteger)plotIndex;
```

Comment:
I'm not sure if the proper term is `axis`. One could think that a graph has the *X* and *Y axes* as in `numberOfDivisionsInXAxisForGraphChartView:`.

How about `numberOfDataPointsForPlotIndex:`?